app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,10 @@
+<% provide(:title, ".about") %>
+<h1><%= t ".about"%></h1>
+<p>
+  The <a href="http://www.railstutorial.org/"><em><%= t ".ruby"%></em></a> is a
```

Comment:
em nên sử dụng i18n cho cả đoạn text luôn nhé, không phải khai báo i18n từng đoạn text như này thì sẽ bị sai ngữ nghĩa do thứ tự danh từ động từ khác nhau ở mỗi ngôn ngữ nhé.
Ví dụ khai báo i18n như sau:
content: "The <a href="http://www.railstutorial.org/">Ruby<em></a> is a ......"